src/main/java/com/rtb/projectmanagementtool/user/AuthOps.java

```diff
@@ -1,9 +1,8 @@
-package com.rtb.projectmanagementtool.auth;
```

Comment:
I would suggest keeping this in its own auth package

Reply:
Only issue is I'll have trouble accessing/changing Skills later...

Reply:
See @godsfavours's PR:  https://github.com/googleinterns/step171-2020/pull/58

He's trying to setup enums so it'll be accessible publicly.  I had just sent him this example:  https://stackoverflow.com/questions/6280937/java-how-to-access-inner-enum-class